SygicUIKit/Classes/Utils/SYUIOrientationUtils.swift

```diff
@@ -0,0 +1,57 @@
+
+public enum Orientation {
```

Comment:
tieto enumy a classy by asi tiez mali mat SYUI prefix

Reply:
jj, v ZoomView som to tak spravil na tieto som zabudol, prepisem to vsetko